### PR TITLE
Add --include-cluster-resources flag for restores

### LINKED ISSUE #136: implement --include-cluster-resources for restores
#128 implemented this flag for backups; this is useful for restores also. Implement.

### PR DESCRIPTION
**Pull request by skriss**
Fixes #136

### CODE REVIEW

File: pkg/restore/restore_test.go [hunk 1]
```diff
@@ -394,6 +401,24 @@ func TestRestoreResourceForNamespace(t *testing.T) {
 			restorers:     map[schema.GroupResource]restorers.ResourceRestorer{schema.GroupResource{Resource: "foo-resource"}: newFakeCustomRestorer()},
 			expectedObjs:  toUnstructured(newTestConfigMap().WithArkLabel("my-restore").ConfigMap),
 		},
+		// NEW
```
Comment (ncdc) [inline pkg/restore/restore_test.go]: Remove

**Comment by ncdc** [comment]
@skriss I was just thinking that if you're creating a `Restore` by hand in yaml/json and you omit `includeClusterResources`, it will default to false and they won't be included. Do you think it would make more sense to change this to `excludeClusterResources` so the default behavior is to include them?

**Comment by ncdc** [comment]
Or make it an optional bool (pointer)

**Comment by skriss** [comment]
Yeah, I think defaulting to include makes sense. I think I like using an optional bool that defaults to true if not specified better; it's more consistent with the corresponding flag on backups.

**Comment by ncdc** [comment]
Yeah, let's go with the optional bool that defaults to true.

**Pull request merged**